client/ipa-client-automount

```diff
@@ -376,11 +377,10 @@ def configure_nfs(fstore, statestore):
                      rpcgssd.service_name, str(e))
 
 def main():
+    check_client_configuration()
```

Comment:
Here you need to handle the exception with
```
try:
  check_client_configuration()
except ScriptError as e:
  sys.exit(e)
```

otherwise the tool exits with a traceback.